src/lib/dateRangeHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import { DateTime } from 'luxon';
+
+export const getTwelveMonthReportDateRange = (): {
+  startDate: string;
+  endDate: string;
+} => {
+  const startDate = DateTime.now()
+    .startOf('month')
+    .minus({ months: 12 })
+    .toISODate();
+  const endDate = DateTime.now()
+    .startOf('month')
+    .minus({ days: 1 })
+    .endOf('day')
+    .toISODate();
+  return { startDate, endDate };
```

Comment:
Personally, I would have this return the formatted string, but it's not required.

```suggestion
  return `${startDate}...${endDate}`;
```